backend/apps/web/models/auths.py

```diff
@@ -86,17 +97,24 @@ def insert_new_auth(
             return None
 
     def authenticate_user(self, email: str, password: str) -> Optional[UserModel]:
-        print("authenticate_user")
-
-        auth = self.table.find_one({"email": email, "active": True})
-
-        if auth:
-            if verify_password(password, auth["password"]):
-                user = self.db.users.find_one({"id": auth["id"]})
-                return UserModel(**user)
+        print("authenticate_user", email)
+        try:
+            auth = Auth.get(Auth.email == email, Auth.active == True)
+            print(auth.email)
+
+            if auth:
+                print(password, str(auth.password))
```

Comment:
thoughts: Should we avoid logging or only log them in debug mode, at least for the passwords?

Reply:
Good catch, forgot about it 😅 It's been removed!